.github/workflows/cyclonus-netpol-test.yaml

```diff
@@ -0,0 +1,114 @@
+name: Cyclonus network policy test
+
+on:
+  # TODO remove push after it runs successfully
```

Comment:
TODO remove push after it runs successfully

Reply:
I assume this can be removed now?

Reply:
👍